rust/sbd-e2e-crypto-client/src/test.rs

```diff
@@ -174,149 +168,33 @@ async fn max_msg_size() {
         (20_000, false),
         (19_968, false),
         (19_952, false),
-        (19_951, true),
+        (19_951, false),
+        (19_950, true),
     ] {
-        assert_eq!(is_ok, c2.send(c1.pub_key(), &MSG[..size]).await.is_ok())
+        let res = c2.send(c1.pub_key(), &MSG[..size]).await;
+        assert_eq!(is_ok, res.is_ok())
     }
 
     let (_, r) = r1.recv().await.unwrap();
-    assert_eq!(&MSG[..19_951], r.as_slice());
+    assert_eq!(&MSG[..19_950], r.as_ref());
```

Comment:
We added a byte to the overhead, so the max message sizes correspondingly shrink by 1